bin/db-restore

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# bin/restore: Restore the local database from staging/production.
+
+set -e
+
+ENVIRONMENT_NAME="${1:-staging}"
+environments=(pentest prod staging)
+
+# Check `ENVIRONMENT_NAME` is one we expect
+if ! [[ ${environments[*]} =~ $ENVIRONMENT_NAME ]]; then
```

Comment:
This condition might have unintended behaviour since it checks whether the string `"pentest prod staging"` is a regex match for the string in `ENVIRONMENT_NAME`. So for example running `db-restore ent` or `db-restore '.*'` both trigger this condition.